python/flask/hello_error.html

```diff
@@ -0,0 +1 @@
+<h1 style="color:#00ffff">Hello error! 👋🏻</h1>
```

Comment:
IMO, this should be hello_world or something like it, to avoid confusion if that makes sense